testinfra/modules/user.py

```diff
@@ -133,6 +133,34 @@ def expiration_date(self):
             epoch = datetime.datetime.utcfromtimestamp(0)
             return epoch + datetime.timedelta(days=int(days))
 
+    @property
+    def get_all_users(self):
+        """Returns a list of local and remote user names
+
+        >>> host.user().get_all_users
+        ["root", "bin", "daemon", "lp", <...>]
+        """
+        all_users = [
+            line.split(":")[0]
+            for line in self.check_output("getent passwd").splitlines()
```

Comment:
I think we could use `[pwd](https://docs.python.org/3.13/library/pwd.html#module-pwd)`? 
```
import pwd
all_users = [pw.pw_name for pw in pwd.getpwall()]
```

Reply:
Unfortunately, this is not possible because testinfra only executes Unix commands on the remote host.